CMakeLists.txt

```
@@ -18,7 +18,7 @@ project(quicr
         LANGUAGES CXX)
 
 option(LINT "Perform linting with clang-tidy" OFF)
-
+option(QUICR_BUILD_SHARED "Build quicr as a SHARED library" OFF)
```

I think general cmake convention is to use lowercase names:
```suggestion
option(quicr_BUILD_SHARED "Build quicr as a SHARED library" OFF)
```

---

In this case I copied the format of the other options above, but maybe we can go change all of them.